pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableImplV2V3.java

```diff
@@ -61,12 +65,15 @@
   private final byte[] _variableSizeDataBytes;
   private final ByteBuffer _variableSizeData;
   private final Map<String, String> _metadata;
+  // Only V3 has _positionalData
+  private final String[] _positionalData;
```

Comment:
I would suggest calling this footer and add some comments on the structure of footer. Please give some example as well.

Reply:
Also update the javadocs in class DataTableBuilder because that's where the structure of the file is listed